### ISSUE TITLE
Paycoin QT wallet crashes on quit

### ISSUE THREAD

**Issue by mdpfeiffer** [PaycoinFoundation/paycoin#301]
Paycoin QT wallet crashes on quit. This occurs when closed using command-q and also when closed from menu bar.

Not sure if this is a problem.

**Comment by IngCr3at1on** [comment]
What OS?

**Comment by mdpfeiffer** [comment]
Mac OS X El Capitan 10.11.1 (15B42)

**Comment by mitchellcash** [comment]
I also experience this on occasion on Mac OS X. Definitely not a 100% occurrence but it does happen. Will need to see if I can find a pattern of occurrence.

**Comment by IngCr3at1on** [comment]
@mdpfeiffer is this a silent crash or does it say anything?

**Comment by mitchellcash** [comment]
I don't believe this has happened to me since 0.3.3.0. @mdpfeiffer can you also confirm that?

**Comment by mdpfeiffer** [comment]
This continues to crash for me on Paycoin 0.3.3.0 running over OS X 10.11.3
The app itself doesn't give a message, but I get a system message that the app has unexpectedly quit.

I'm not sure if that answers your question about if this is "a silent crash or does it say anything?"

**Comment by mdpfeiffer** [comment]
Here's a portion o the crash report:

Process:               Paycoin [42450]
Path:                  /Applications/Paycoin.app/Contents/MacOS/Paycoin
Identifier:            com.yourcompany.Paycoin
Version:               ???
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           Paycoin [42450]
User ID:               501

Date/Time:             2016-01-28 23:58:44.343 -0500
OS Version:            Mac OS X 10.11.3 (15D21)
Report Version:        11
Anonymous UUID:        Redacted

Sleep/Wake UUID:       Redacted

Time Awake Since Boot: 260000 seconds
Time Since Wake:       1900 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

[…]

Error Code:      0x00000004
Trap Number:     14
